public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -731,60 +731,60 @@ block style-property-name-dart-diff
 +makeExample('template-syntax/ts/app/app.component.html', 'event-binding-1')(format=".")
 :marked
   ### Target event
-  A **name between enclosing parentheses** &mdash; for example, `(click)` &mdash;
+  A **name between parentheses** &mdash; for example, `(click)` &mdash;
   identifies the target event. In the following example, the target is the button’s click event.
 +makeExample('template-syntax/ts/app/app.component.html', 'event-binding-1')(format=".")
 :marked
-  Some people prefer the `on-` prefix alternative, known as the *canonical form*:
+  Some people prefer the `on-` prefix alternative, known as the **canonical form**:
 +makeExample('template-syntax/ts/app/app.component.html', 'event-binding-2')(format=".")
 :marked
-  Element events may be the more common targets, but Angular looks first to see if the name matches an event property
+  Element events may be the more common targets, but Angular looks first to see if the name matches the event property
   of a known directive, as it does in the following example:
 +makeExample('template-syntax/ts/app/app.component.html', 'event-binding-3')(format=".")
 
 .l-sub-section
   :marked
-    The `myClick` directive is further described below in the section
-    on [Aliasing input/output properties](#aliasing-io).
+    The `myClick` directive is further described in the section
+    on [aliasing input/output properties](#aliasing-io).
 
 :marked
-  If the name fails to match an element event or an output property of a known directive,
+  If the name fails to match the element event or output property of a known directive,
   Angular reports an “unknown directive” error.
 
   ### *$event* and event handling statements
-  In an event binding, Angular sets up an event handler for the target event.
+  In event binding, Angular sets up an event handler for the target event.
 
   When the event is raised, the handler executes the template statement.
-  The template statement typically involves a receiver that wants to do something
-  in response to the event, such as take a value from the HTML control and store it
-  in a model.
+  The template statement typically involves a receiver, which performs an action
+  in response to the event, such as storing a value from the HTML control
+  into a model.
 
   The binding conveys information about the event, including data values, through
-  an **event object named `$event`**.
+  an **event object named** `$event`.
```

Comment:
I did this because it's rare to see bolded code. Making it a code block already makes it stand out from the rest of the text, and it's rare that you will see a bolded piece of code. It's also not consistent with the rest of the document (at least the parts that I read).